### DISCUSSION TITLE
GitHub sponsors

### DISCUSSION THREAD

**Discussion by thomaseizinger** [Radiokot/photoprism-android-client#49] [Q&A]
Are you open to setting up GitHub sponsors? I'd like to support your work here and GitHub sponsors makes it really easy to do so! :)

**Comment by Radiokot** [comment]
Hey, Thomas! 
My account is not eligible for GitHub sponsorship for whatever reasons. 
I'm thankful for your will to support my work, I think you could find another donation method that works for you on my [tip page](https://radiokot.com.ua/tip). If you'd like to make a public donation, I'm pleased to include you to the project description 🙌

**Reply by thomaseizinger** [reply]
Hey! Thanks, I'll pick one of the other donation options :)

No need for a mention.

**Reply by Radiokot** [reply]
Thank you very much for supporting the project, Thomas!
